### PR TITLE
Home activity plus SeatReservation hotfix

### PR DESCRIPTION
**Pull request by alexaib2002**
This PR brings a barebones home screen activity (though it should be enough for the usecases of this app).
For not creating another PR, I've also included a hotfix for fixing a bug caused in the latest merge.

### CODE REVIEW

File: app/CineHub/app/src/main/java/org/cinehub/HomeActivity.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+
+        TextView tvHomeTitle = findViewById(R.id.tvHomeTitle);
+        TextView tvHomeSub = findViewById(R.id.tvHomeSub);
+        FloatingActionButton fabAddTicket = findViewById(R.id.fabAddTicket);
+
+        User user = getIntent().getParcelableExtra(LoginActivity.EXTRA_USER);
+        CinehubAPI api = new CinehubAPI();
```
Comment (Jkutkut) [inline app/CineHub/app/src/main/java/org/cinehub/HomeActivity.java]: This is not the way to interact with the API. The use of the interfaces should be the only way to interact with it

File: app/CineHub/app/src/main/java/org/cinehub/ReservationItem.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View v = inflater.inflate(R.layout.fragment_reservation_item, container, false);
+        ((TextView) v.findViewById(R.id.tvResItemPos))
+                .setText(getString(R.string.label_seat_placement,
+                        reservation.getRow(), reservation.getCol()));
+        CinehubAPI api = new CinehubAPI();
```
Comment (Jkutkut) [inline app/CineHub/app/src/main/java/org/cinehub/ReservationItem.java]: Same problem here

**Review by Jkutkut — CHANGES_REQUESTED** [review]
On the SeatReservation class, the addition of a aux constructor was made. The use of that constructor should be replaced with the use of the class Seat.

```java
public SeatReservation(int row, int col) {
    this(-1, row, col, -1);
}
```

**Comment by Jkutkut** [comment]
A PR with changes to the API is going to be made to this branch to enforce the correct use of the interfaces. Please, read the wiki before using the API.

**Comment by alexaib2002** [comment]
> A PR with changes to the API is going to be made to this branch to enforce the correct use of the interfaces. Please, read the wiki before using the API.

Your PR have been merged. Feel free to point out any other issue

**Review by Jkutkut — APPROVED** [review]
All OK

**Pull request merged**